cms/envs/production.py

```diff
@@ -545,3 +545,14 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+############## Settings for assets upload ##############
+
+### Size of chunks into which asset uploads will be divided
+UPLOAD_CHUNK_SIZE_IN_MB = ENV_TOKENS.get('UPLOAD_CHUNK_SIZE_IN_MB', UPLOAD_CHUNK_SIZE_IN_MB)
+
+### Max size of asset uploads to GridFS
+MAX_ASSET_UPLOAD_FILE_SIZE_IN_MB = ENV_TOKENS.get(
+    'MAX_ASSET_UPLOAD_FILE_SIZE_IN_MB',
+    MAX_ASSET_UPLOAD_FILE_SIZE_IN_MB,
```

Comment:
@diegomillan this function does not receive more parameters, maybe it can be left without a trailing comma

Reply:
You are right @luismorenolopera ... done